examples/nek/bps/bps.usr

```diff
@@ -18,6 +18,16 @@ C> @file
 C> Mass and diffusion operators examples using Nek5000
 C TESTARGS -c {ceed_resource} -e bp1 -n 1 -b 4 -test
 
+C PragmaOMP preprocessor directive
+#ifndef CeedPragmaOMP
+#  ifdef _OPENMP
+#    define CeedPragmaOMP_(a) _Pragma(#a)
+#    define CeedPragmaOMP(a) CeedPragmaOMP_(omp a)
+#  else
+#    define CeedPragmaOMP(a)
+#  endif
+#endif
```

Comment:
I'm pretty sure Fortran won't be defining `_OPENMP`, so these will always be inactive. But if your Fortran does define that macro, you'll get a syntax error because Fortran doesn't understand `_Pragma`. OpenMP syntax is different for Fortran (directives go in comments, `!$omp simd` or `C$omp simd`).

Reply:
Thanks! I have updated it now and checked that `gfortran -cpp -E -dM -fopenmp` gives `#define CeedPragmaOMP(a) CeedPragmaOMP_(omp a)` in the list of defines.
However, I cannot test it at the moment as I never got to install Nek.

Reply:
It'll fail because the string is being quoted (needed for `_Pragma`). This test works for me in isolation:
```c
#ifndef CeedPragmaOMP
#  ifdef _OPENMP
#    define CeedPragmaOMP(a) !$omp a
#  else
#    define CeedPragmaOMP(a)
#  endif
#endif
```

Reply:
Fixed. Thanks!

Reply:
@jed I have now updated the Nek example to the SIMD version. (I have found this reference https://gcc.gnu.org/onlinedocs/gfortran/IVDEP-directive.html).
Again, I cannot test this at the moment. I think it should be good.